utils/build.ps1

```diff
@@ -1622,6 +1623,7 @@ function Build-Runtime([Platform]$Platform, $Arch) {
         SWIFT_ENABLE_EXPERIMENTAL_STRING_PROCESSING = "YES";
         SWIFT_ENABLE_SYNCHRONIZATION = "YES";
         SWIFT_ENABLE_VOLATILE = "YES";
+        SWIFT_SHOULD_BUILD_EMBEDDED_STDLIB_CROSS_COMPILING = "YES";
```

Comment:
This is a problem - we are going to build the same embedded standard libraries 7 times. We should extract a special build for the embedded runtimes and distribute that as a separate SDK.

Reply:
FWIW, this matches what we're doing with the other platforms. I'm investigating how we can do this with SDKs but people are asking for Windows support for embedded now.

Reply:
IMO yes these should be a separate pieces, and ideally not even as an "SDK" but delivered as source that SwiftPM can build on demand for the user's specific flags. Until then however I think including (some) copies of Swift.swiftmodule for embedded windows.